lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -79,9 +79,7 @@ end
 /******************************************************************************/
 
 e2function entity entity(id)
-	local ent = ents.GetByIndex(id)
-	if not IsValid(ent) then return nil end
-	return ent
+	return Entity(id) -- ents.GetByIndex is alias of _G.Entity function. IsValid check is redundant here.
```

Comment:
If the entity does not exist, it will return GLua `NULL` (constant) - not Lua `nil` value.